### PR TITLE
[1.3] CMake For maintenance branch 1.3.x

### PR DESCRIPTION
**Pull request by Karm**
We haven't released anything in quite a while and people need an easy way to compile themselves.
CMake files do not affect current build environment in nay way and the commit doesn't touch code, so it is safe to accept even into a maintenance branch.

(I remember that CMake PR for 1.3.x was rejected once and moved to master, so this is a second attempt.)

An example of such demand: https://developer.jboss.org/thread/272671

### CODE REVIEW

**Comment by rhusar** [comment]
I don't see the harm and the added value is for community to build this easily.

Before merging, please add the Jira number to the commit message as usual (I guess its https://issues.jboss.org/browse/MODCLUSTER-465?) and synchronize the 1.2.x version (https://github.com/modcluster/mod_cluster/pull/203)

File: native/cmake/modules/FindAPACHE.cmake [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#
+#  APACHE_FOUND         - System has APACHE
+#  APACHE_INCLUDE_DIR   - The APACHE include directory
+#
+#  APACHE_LOCATION      - setting this enables search for apache libraries / headers in this location
+
+#
+# Include directories
+#
+find_path(APACHE_INCLUDE_DIR
```
Comment (rhusar) [inline native/cmake/modules/FindAPACHE.cmake]: The other scripts are using ALL CAPS, is this intentional?

Comment (Karm) [inline native/cmake/modules/FindAPACHE.cmake]: @rhusar  This is case insensitive, it doesn't matter. Further iteration will probably put everything to all caps as that is what I can see in httpd's files....   Let's not block the PR on this.

**Comment by Karm** [comment]
@rhusar : Ready to be merged. Amended commit message, added missing newlines. Tested that is builds both headless nmake and also Visual Studio solution.

File: native/cmake/modules/FindAPACHE.cmake (hunk 1 above)
Comment (rhusar) [inline native/cmake/modules/FindAPACHE.cmake]: Sure, no problem. Ideally this file can be extended to support building on macOS out of box (wip).

**Comment by rhusar** [comment]
Merging.

I will add the missing readme in a follow up commit.

**Pull request merged**